rosdep/base.yaml

```diff
@@ -1509,6 +1511,10 @@ libgazebo5-dev:
   fedora: [gazebo-devel]
   gentoo: [sci-electronics/gazebo]
   ubuntu: [libgazebo5-dev]
+libgazebo7-dev:
+  arch: [gazebo]
+  gentoo: [sci-electronics/gazebo]
```

Comment:
The Gentoo friends are updated with [gazebo versions](https://packages.gentoo.org/packages/sci-electronics/gazebo). It lives in the unstable amd64 branch (note the ~amd64 mark).

The Arch Linux friends are a bit behind but also using [gazebo7 versions](https://aur.archlinux.org/packages/gazebo/).

These distributions don't add a numbered version to the package but both have gazebo7 available.